BDArmory/Competition/BDACompetitionMode.cs

```diff
@@ -1017,6 +1017,7 @@ public void OnVesselModified(Vessel vessel)
             if (VesselModuleRegistry.ignoredVesselTypes.Contains(vessel.vesselType)) return;
             if (!BDArmorySettings.AUTONOMOUS_COMBAT_SEATS) CheckForAutonomousCombatSeat(vessel);
             if (BDArmorySettings.DESTROY_UNCONTROLLED_WMS) CheckForUncontrolledVessel(vessel);
+            if (BDArmorySettings.COMPETITION_GM_KILL_WEAPON || BDArmorySettings.COMPETITION_GM_KILL_ENGINE || (BDArmorySettings.COMPETITION_GM_KILL_HP > 0)) CheckForGMCulling(vessel);
```

Comment:
This should first check for `BDArmorySettings.COMPETITION_GM_KILL_TIME > 0`.

Reply:
Only if GM kill toggles are intended to *only* kill after a delay, and can't do instantaneous culling (so the timer serves as a second I|0 toggle to the individual toggles), or 'Off' = -1 instead of 0 in BDArmorySetup.

Reply:
I think having `"Off" = -1` (as you've currently got it) is good for disabling all of them at once.